src/ngraph/runtime/cpu/mkldnn_emitter.hpp

```diff
@@ -391,6 +392,85 @@ namespace ngraph
                                                 const mkldnn::memory::desc& dweights_desc,
                                                 const double eps);
 
+                template <typename OP>
+                size_t build_rnn(const ngraph::Node* node,
```

Comment:
looks good. but dont see a need for templatized method.

Reply:
ya but `OP` can be either `ngraph::op::Lstm` or `ngraph::op::Rnn` so I templatized it